test/e2e/diff_test.go

```diff
@@ -130,42 +131,46 @@ RUN echo test
 		Expect(session.OutputToStringArray()).To(BeEmpty())
 	})
 
-	It("podman diff container and image with same name", func() {
-		imagefile := "/" + stringid.GenerateRandomID()
-		confile := "/" + stringid.GenerateRandomID()
-
-		// Create container image with the files
-		containerfile := fmt.Sprintf(`
-FROM  %s
-RUN touch %s`, ALPINE, imagefile)
-
-		name := "podman-diff-test"
-		podmanTest.BuildImage(containerfile, name, "false")
-
-		session := podmanTest.Podman([]string{"run", "--name", name, ALPINE, "touch", confile})
-		session.WaitWithDefaultTimeout()
-		Expect(session).Should(ExitCleanly())
-
-		// podman diff prefers image over container when they have the same name
-		session = podmanTest.Podman([]string{"diff", name})
-		session.WaitWithDefaultTimeout()
-		Expect(session).Should(ExitCleanly())
-		Expect(session.OutputToStringArray()).To(HaveLen(2))
-		Expect(session.OutputToString()).To(ContainSubstring(imagefile))
-
-		session = podmanTest.Podman([]string{"image", "diff", name})
-		session.WaitWithDefaultTimeout()
-		Expect(session).Should(ExitCleanly())
-		Expect(session.OutputToStringArray()).To(HaveLen(2))
-		Expect(session.OutputToString()).To(ContainSubstring(imagefile))
-
-		// container diff has to show the container
-		session = podmanTest.Podman([]string{"container", "diff", name})
-		session.WaitWithDefaultTimeout()
-		Expect(session).Should(ExitCleanly())
-		Expect(session.OutputToStringArray()).To(HaveLen(2))
-		Expect(session.OutputToString()).To(ContainSubstring(confile))
-	})
+	// Commented out on July 23, 2025 to avoid issue noted in
```

Comment:
In the interested of time I am going to accept that but please do not ever comment out tests like that.
This will make the git blame much harder follow, it will make the test invisible to logs (a skip will still be logged), and the skip FIXME checker will not find it once the linked issue gets closed.

The proper way is to add a skip
```
Skip("FIXME: #26680 - diff test showing different behaviour depending on the storage driver")
```